src/lib/sort/sort-header.scss

```diff
@@ -72,6 +73,7 @@ $mat-sort-header-arrow-transition: 225ms cubic-bezier(0.4, 0, 0.2, 1);
   height: $mat-sort-header-arrow-thickness;
   transition: $mat-sort-header-arrow-transition;
   position: absolute;
+  top: 0rem;
```

Comment:
`0rem` is the same as just `0`.

Reply:
Oh goodness, I think this was auto-completed IntelliJ. Thanks for catching that